docker-compose.yaml

```diff
@@ -2,12 +2,14 @@ version: "3.8"
 services:
 
   backend:
-    image: hmctspublic.azurecr.io/backstage/backend:prod-03bc263
+    image: docker.io/library/backstage
```

Comment:
This right?

Reply:
ftr it's just `backstage` on your docker cli, docker buildx outputs the full path when you build with it though and i just copy pasted it in when testing